document_test.go

```diff
@@ -198,15 +235,13 @@ func TestDocument_RenderAndReload_ChangeCheck_Stripe(t *testing.T) {
 	tc := compReport.TotalChanges()
 	bc := compReport.TotalBreakingChanges()
 	assert.Equal(t, 0, bc)
-	assert.Equal(t, 820, tc)
+	assert.Equal(t, 819, tc)
```

Comment:
Do you know what changed? (pun intended)

Reply:
Yeah so a lot of these changes came from the fact I am now more correctly retaining the original styling of the document when it does a round trip through libopenapi, a few things were being picked up as changes just because they were being rendered out differently to how they were originally